src/main/kotlin/app/revanced/patches/youtube/ad/infocardsuggestions/patch/HideInfocardSuggestionsPatch.kt

```diff
@@ -61,6 +54,21 @@ class HideInfocardSuggestionsPatch : BytecodePatch(
             invoke-static {p1}, Lapp/revanced/integrations/patches/HideInfoCardSuggestionsPatch;->hideInfoCardSuggestions(Landroid/view/View;)V
         """)
 
+        // hide the header creation to prevent the info card to pop up for a second
```

Comment:
Why is it necessary in addition to this, to hide the view in the method which is resolved from HideInfocardSuggestionsFingerprint?

```suggestion
        // prevent showing the info card header to pop up for a second
```

Reply:
> Why is it necessary in addition to this, to hide the view in the method which is resolved from HideInfocardSuggestionsFingerprint?

https://www.reddit.com/r/revancedapp/comments/yo9k2e/info_card_doesnt_hide_perfectly/

Reply:
Meanwhile, HideInfocardSuggestionsFingerprint it's needed to hide suggestion cards in incognito mode.

Reply:
Please name the fingerprints accordingly then.